core/sql/optimizer/ObjectNames.cpp

```diff
@@ -1385,6 +1386,17 @@ void QualifiedName::print(FILE* ofd, const char* indent, const char* title) cons
 #endif
 }
 
+NABoolean QualifiedName::isHistograms() const
+{
+   return (getObjectName() == "SB_HISTOGRAMS");
+}
+
+NABoolean QualifiedName::isHistogramIntervals() const
+{
+   return (getObjectName() == "SB_HISTOGRAM_INTERVALS");
+}
```

Comment:
Might want to use the predefined values found in ComSmallDefs.h - HBASE_HIST_NAME & HBASE_HISTINT_NAME